string ssid
string password
---
bool res
string message
